File: v2/csharp/src/main/csharp/io/swagger/Model/PostResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Postresponse {
    

    
    public Post Post { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Postresponse {\n");
      
      sb.Append("  Post: ").Append(Post).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/Postresponse.java
package vestorly.client.model;

import vestorly.client.model.Post;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Postresponse  {
  
  private Post post = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("post")
  public Post getPost() {
    return post;
  }
  public void setPost(Post post) {
    this.post = post;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Postresponse {\n");
    
    sb.append("  post: ").append(post).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
